iptc/ip6tc.py

```diff
@@ -273,7 +273,7 @@ def _create_mask(self, plen):
             if plen >= 8:
                 mask.append(0xff)
             elif plen > 0:
-                mask.append(2 ** plen - 1)
+                mask.append(0xff>>(8-plen)<<(8-plen))
```

Comment:
We need the top `plen` bits to be `1`, so I think that's `0xff ^ (2 ** (8-plen) - 1)`?
This is just dropping the bottom `8-plen` bits though, so the double shift seemed a bit more obvious.